python/desc/imsim/imSim_class_factory.py

```diff
@@ -52,8 +57,16 @@ def imSim__init__(self, phosim_objects, obs_metadata, catalog_db=None):
     self.db_obj = type('DummyDB', (), dict(epoch=2000))
 
     # Add noise and sky background
-    self.noise_and_background = ExampleCCDNoise(addNoise=True,
-                                                addBackground=True)
+    # The simple code using the default lsst-GalSim interface would be:
+    #
+    #    self.noise_and_background = ExampleCCDNoise(addNoise=True,
+    #                                                addBackground=True)
+    #
+    # But, we need a more realistic sky model and we need to pass more than
+    # this basic info to use Peter Y's ESO sky model.
+    # We must pass obs_metadata, chip information etc...
+    self.noise_and_background = ESOSkyModel(obs_metadata, addNoise=True,
```

Comment:
I'm wondering of we should allow `self.noise_and_background` and `self.PSF` to be settable through the constructor, i.e.,
```
def imSim__init__(self, phosim_objects, obs_metadata, catalog_db=None, noise_and_background=None, PSF=None):
```
and set them to the values here if the default `None` values are passed.